package.json

```diff
@@ -11,20 +11,29 @@
   "bin": {
     "uuid": "./bin/uuid"
   },
+  "sideEffects": false,
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
     "eslint": "4.5.0",
     "mocha": "3.1.2",
-    "runmd": "0.1.7"
+    "runmd": "^1.0.1"
   },
   "scripts": {
+    "build": "babel src --out-dir lib",
+    "pretest": "npm run build",
     "test": "mocha test/test.js",
     "md": "runmd --watch --output=README.md README_js.md",
-    "prepare": "runmd --output=README.md README_js.md"
+    "prepare": "npm run build && runmd --output=README.md README_js.md"
   },
   "browser": {
     "./lib/rng.js": "./lib/rng-browser.js",
     "./lib/sha1.js": "./lib/sha1-browser.js",
-    "./lib/md5.js": "./lib/md5-browser.js"
+    "./lib/md5.js": "./lib/md5-browser.js",
+    "./src/rng.js": "./src/rng-browser.js",
```

Comment:
duplicate?

Reply:
Nope, this is intentional - i rewrite here imports for both types of modules. If you use node or "cjs bundler" you will end up with files from `lib`, if you use module aware bundler like webpack2+ you will end up with files from `src`

Reply:
Those are not duplicates, there are 3 "rewrite" keys from lib to lib directory and 3 "rewrite" keys from src to src. This package has **legacy** `index.js` - it actually has no main/module fields, because you are supposed to "cherry-pick" what you want by requiring/importing i.e. `uuid/v4`

The resolution algorithms works like this:
- `uuid` directory is found
- `v4` directory is found
- as this directory has only `package.json` (since this PR) with `main` and `module` fields they are followed regarding regular rules, node will follow `main`, webpack etc will follow `module`
- appropriate directory is visited (let's say we use webpack so its `src`) and for each requested file it is checked in "closest" `package.json` (closest from `uuid/src` is the one in the root of the package) if `browser` fields has a "rewrite" from the requested relative path to some other relative path - so webpack will follow this rewrite and use appropriate file which targets browser AND is written using es6 modules syntax